RecoTracker/MkFit/plugins/MkFitOutputConverter.cc

```diff
@@ -110,15 +150,23 @@ class MkFitOutputConverter : public edm::global::EDProducer<> {
   const float qualityMaxZ_;
   const float qualityMaxPosErrSq_;
   const bool qualitySignPt_;
+
+  const std::string tfDnnLabel_;
+  const edm::ESGetToken<TfGraphDefWrapper, TfGraphRecord> tfDnnToken_;
+  const int algo_;
 };
 
 MkFitOutputConverter::MkFitOutputConverter(edm::ParameterSet const& iConfig)
-    : eventOfHitsToken_{consumes<MkFitEventOfHits>(iConfig.getParameter<edm::InputTag>("mkFitEventOfHits"))},
+    : flag_{int(iConfig.getParameter<int>("flag"))},
```

Comment:
It may be better to call this variable with a more 'direct' name, and maybe use a `bool`?